forward_engineering/alterScript/alterScriptHelpers/containerHelpers/alterSequenceHelper.js

```diff
@@ -0,0 +1,91 @@
+const { AlterScriptDto } = require('../../types/AlterScriptDto');
+const { App } = require('../../../types/coreApplicationTypes');
+const sequencesCompModKey = 'sequences';
+
+/**
+ * @param {{ app: App }}
+ * @return {({ container, dbVersion }: { container: object, dbVersion: string }) => AlterScriptDto[]}
+ * */
+const getAddContainerSequencesScriptDtos = ({ app }) => ({ container, dbVersion }) => {
+    const _ = app.require('lodash');
+    const ddlProvider = require('../../../ddlProvider')(null, { dbVersion }, app);
+    const { getDbName } = require('../../../utils/general')(_);
+    const schemaName = getDbName([container.role]);
+
+    return (container.role?.sequences || [])
+        .map((sequence) => ddlProvider.createSchemaSequence({ schemaName, sequence }))
+        .map((script) => AlterScriptDto.getInstance([script], true, false))
```

Comment:
do we really have to iterate twice here? Can't we just expand the function block of a first `map` and do whatever we need within a single iteration?